griptape/tasks/image_query_task.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from typing import Callable
+
+from attr import define, field
+
+from griptape.artifacts import ImageArtifact, TextArtifact
+from griptape.engines import ImageQueryEngine
+from griptape.tasks import BaseTask
+from griptape.utils import J2
+
+
+@define
+class ImageQueryTask(BaseTask):
+    image_query_engine: ImageQueryEngine = field(kw_only=True)
```

Comment:
May want to merge in #591 first so that we can do something similar to [this](https://github.com/griptape-ai/griptape/pull/591/files#diff-30ce1ef1ca99f2869a727dd437c0a9985d34cddc1e9a922142bdf4020bd366bfR49)